sqlite/sqlite.go

```diff
@@ -21,13 +21,13 @@ const (
 // SqlStore is a wrapper around the db and provides basic functionality for maintaining the db
 // including flushing the data from the db during end-to-end testing.
 type SqlStore struct {
-	mu  sync.Mutex
-	db  *sql.DB
+	Mu  sync.Mutex
+	DB  *sqlx.DB
```

Comment:
Needed to export these so that they can be used by the services utilizing the database in different packages.